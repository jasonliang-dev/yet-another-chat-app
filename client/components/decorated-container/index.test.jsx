import React from 'react';
import { mount } from 'enzyme';
import DecoratedContainer from './index';

describe('<DecoratedContainer />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<DecoratedContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
