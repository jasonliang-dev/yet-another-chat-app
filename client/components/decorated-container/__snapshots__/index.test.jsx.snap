// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DecoratedContainer /> should match snapshot 1`] = `
<DecoratedContainer>
  <DottedBackground
    className="min-h-screen flex flex-col"
    style={Object {}}
  >
    <div
      className="min-h-screen flex flex-col"
      style={
        Object {
          "backgroundImage": "url(test-file-stub)",
        }
      }
    >
      <div
        className="w-full h-2 bg-teal-500"
      />
      <div
        className="flex-grow flex items-center justify-center"
      />
    </div>
  </DottedBackground>
</DecoratedContainer>
`;
